helm_deploy/create-and-vary-a-licence-api/values.yaml

```diff
@@ -37,6 +37,36 @@ generic-service:
       DB_NAME: "database_name"
       DB_USER: "database_username"
       DB_PASS: "database_password"
+    notify-templates:
+      NOTIFY_TEMPLATES_VARIATIONFORAPPROVAL: "VARIATION_FOR_APPROVAL"
+      NOTIFY_TEMPLATES_LICENCEAPPROVED: "LICENCE_APPROVED"
+      NOTIFY_TEMPLATES_INITIALLICENCEPROMPT: "INITIAL_LICENCE_PROMPT"
+      NOTIFY_TEMPLATES_URGENTLICENCEPROMPT: "URGENT_LICENCE_PROMPT"
+    pdu-head-config:
+      PDU_CONTACTS_WPTNWS_FORENAME: "WPTNWS_FORENAME"
+      PDU_CONTACTS_WPTNWS_SURNAME: "WPTNWS_SURNAME"
+      PDU_CONTACTS_WPTNWS_DESCRIPTION: "WPTNWS_DESCRIPTION"
+      PDU_CONTACTS_WPTNWS_EMAIL: "WPTNWS_EMAIL"
+      PDU_CONTACTS_WPTWES_FORENAME: "WPTWES_FORENAME"
+      PDU_CONTACTS_WPTWES_SURNAME: "WPTWES_SURNAME"
+      PDU_CONTACTS_WPTWES_DESCRIPTION: "WPTWES_DESCRIPTION"
+      PDU_CONTACTS_WPTWES_EMAIL: "WPTWES_EMAIL"
+      PDU_CONTACTS_WPTGWT_FORENAME: "WPTGWT_FORENAME"
+      PDU_CONTACTS_WPTGWT_SURNAME: "WPTGWT_SURNAME"
```

Comment:
You have put these secrets in `create-and-vary-a-licence-dev` instead of `create-and-vary-a-licence-api-dev`... wrong namespace?

Reply:
Also, real life people should not have had their contact details added as secrets to dev environment?

Reply:
Aha. Yes you're right. I'll moved those now. We won't actually send to these people in dev or preprod - the Notify account is hobbled to send only to members of our team. But its a way to check the right values are present.

Reply:
Fixed this now.